usaspending_api/awards/v2/filters/location_filter_geocode.py

```diff
@@ -264,10 +274,10 @@ def get_award_ids_by_city(scope: str, city: str, country_code: str, state_code:
     }
```

Comment:
Unable to add comment to line 261|271
add .upper()
`{"match": {"{}_city_name.keyword".format(scope): es_sanitize(city).upper()}},`